internal/synthesis/synthesis.go

```diff
@@ -64,7 +64,7 @@ type EntityRule struct {
 	Identifier string      `mapstructure:"identifier"` // Name of the attribute that will be used to uniquely identify the synthesized entity.
 	Name       string      `mapstructure:"name"`       // Name of the attribute that will be used to name the synthesized entity.
 	Conditions []Condition `mapstructure:"conditions"` // List of rules used to determining if a metric belongs to this entity.
-	Tags       Tags        `mapstructure:"tags"`       // List of attributes that will be added to the entity as tags.
+	Tags       Tags        `mapstructure:"tags"`       // List of attributes that will be added to the entity as metadata.
```

Comment:
Should we keep this old name? Changing might be a bit painful but will make everything more clear for our future selves.

Reply:
This structure follows the naming conventions of the entity definition files so I would keep it like this to avoid more confusion since we will be renaming what entity definitions describe in the entity rules. On line 144 there is the actual conversion of naming between entity definitions and infra sdk.
Any way this hole code has a short expectation of live since the hole entity synthesis process will be removed from pomi